dff/context_storages/__init__.py

```diff
@@ -1,5 +1,10 @@
 # -*- coding: utf-8 -*-
 # flake8: noqa: F401
+# fmt: off
```

Comment:
Why do we off formatting ? And Why do not we on it after?

Reply:
It should be just ignored, maybe?

Reply:
We turned it off, because standarts don't allow code before imports, and `nest_asyncio` requires it.  
We didn't turn it on later because this command has file scope.

Reply:
Can we use  line scope commands for this purpose?

Reply:
This is unresolved issue